routes/api.php

```diff
@@ -19,3 +21,6 @@
     Route::post('/login', LoginFarmer::class)->name('login');
     Route::post('/logout', LogoutFarmer::class)->name('logout')->middleware('auth:api');
 });
+
+Route::post('/create.farmer.report', CreateFarmerReport::class);
```

Comment:
Revise `/create.farmer.report` API route. Do not use dots in URLs. Use `/farmer-reports` instead. Check out this [reference](https://usercontent.one/wp/www.kennethlange.com/wp-content/uploads/2020/04/customer_rest_api.png) in how to structure API routes based on the REST standard.

Reply:
Changed the URL.